_tarsnap

```diff
@@ -0,0 +1,163 @@
+#compdef tarsnap
+
+# zsh completion function for tarsnap 1.0.37
```

Comment:
Could you add some hint for zsh users as to how they'd use this file?  i.e. should it be copied into `/usr/share/zsh/completion/` , or `/etc/zsh/`, or `$HOME/.zsh/` ?